mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1065,12 +1065,9 @@ def _generate_model_predictions(self):
             self.is_binomial = self.num_classes <= 2
 
             if self.is_binomial:
-                if list(self.label_list) not in [[0, 1], [-1, 1]]:
```

Comment:
should this logic still execute if pos_label is not specified?

Reply:
I think it's too restrictive in any case.